client/src/views/Tests/TestForm/TestEmails.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import { useAuth } from "../../../hooks";
+import { models } from '@yahalom-tests/common';
+import { FormField } from '../../../components';
+import { EmailForm } from "./EmailForm"
+
+export const TestEmails: React.FC<models.dtos.TestDto> = (testProp) => {
+    const { activeStudyField, buildAuthRequestData } = useAuth();
+    //set test prop as component state.
+    const [test, setTest] = useState<models.dtos.TestDto>({ ...testProp });
```

Comment:
Please use an interface, and get the details with a Pick, as done in other files.
In addition, having the test as a state here is **very bad**, since we don't want this component to handle the test, but only call callbacks if needed.